### ISSUE TITLE
Crash during Kex 5 mission

### ISSUE THREAD

**Issue by Fiachra-Knox** [naev/naev#2135]
On version 0.9.2, select Strangelove's Lab before boarding Dr. Strangelove's ship. Go through the cutscene, then land on Strangelove's Lab to trigger the crash.

**Comment by not-pyroman** [comment]
could you please send logs? See https://github.com/naev/naev/wiki/Bugs#log-location for where they are stored

**Comment by Fiachra-Knox** [comment]
The `stderr.txt` file is blank. `stdout.txt` is as follows:
```
Trying default datapath: /__w/naev/naev/source/extras/windows/installer/bin/dat
Trying default datapath: C:\Program Files (x86)\Steam\steamapps\common\Naev\/dat
Loaded configuration: C:\Users\fiach\AppData\Roaming/naev/conf.lua
Read locations, searched in order:
    C:\Users\fiach\AppData\Roaming\.\naev\
    C:\Program Files (x86)\Steam\steamapps\common\Naev\/dat
Write location: C:\Users\fiach\AppData\Roaming\.\naev\

 Sea of Darkness
Reached main menu
```

**Comment by not-pyroman** [comment]
did you launch the game again after the crash? if so go for the most recent stderr while I get 0.9.2 installed to check myself, Kex 5 you said right?

**Comment by Fiachra-Knox** [comment]
I did launch the game again, but the most recent stderr file apart from `stderr.txt` is over 5 days old, and the stdout files look just like the one I posted.

**Comment by not-pyroman** [comment]
I will have to check later as im out of time right now, sorry

**Comment by not-pyroman** [comment]
That is an oversight, i am not too sure how to fix it nor can i right now but it should be fixed soon, for now just avoid doing that

**Comment by bobbens** [comment]
Could you provide your save file please? I can try to reproduce.

**Comment by bobbens** [comment]
Actually, @not-pyroman mentioned on discord it's probably you having the spob targeted when it gets removed by the unidiff, so it should be a straight-forward fix.

**Comment by bobbens** [comment]
Should be fixed in main / 0.9.3. As a workaround for 0.9.2, don't select the planet when boarding the ship and it shouldn't crash. Thanks!
